views/js/controller/runner/runner.js

```diff
@@ -115,7 +115,7 @@ define([
             var runnerOptions = _.defaults({}, options, defaults);
             var exitReason;
             var $container = $('.runner');
-            var logger     = loggerFactory('controller/runner', { runnerOptions : runnerOptions });
+            var logger     = loggerFactory('controller/runner', { serviceCallId : runnerOptions.serviceCallId });
```

Comment:
Maybe use [lodash to `pick`](https://lodash.com/docs/2.4.2#pick) only a few elements? 

```javascript
loggerFactory('controller/runner', {
    runnerOptions : _.pick(runnerOptions, ['serviceCallId', 'testCompilation', 'testDefinition', 'provider', ]
});
```

Reply:
`provider` might not be usefull, does it ? 
the list of active plugins more, but as a flat list with only the ids of active plugins like : `activePlugins: ['answerCache', 'next', ...]`

Reply:
@krampstudio i have updated the PR. Please review it.
P.S. `testCompilation` and `testDefinition` may be given from `serviceCallId`